pages/deployments/global-services.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Configure Global Services
+description: Deploy services by default on specified clusters.
+---
+
+Global services are automatically replicated across existing and new clusters matching the set of defined specifications.
+
+Once a service is deployed onto any cluster, it can be marked global by clicking the three dots in the deployments table and selecting "Make global".
```

Comment:
@michaeljguarino what's the CLI command for this?

Reply:
not sure we have one yet